src/LibEventLoop.php

```diff
@@ -298,7 +298,7 @@ private function unsubscribeStreamEvent($stream, $flag)
      */
     private function createTimerCallback()
     {
-        $this->timerCallback = function ($_, $_, $timer) {
+        $this->timerCallback = function ($_, $__, $timer) {
```

Comment:
I see why we would want this change, but for the reference only: Afaict this line is currently untested because the libevent extension does not compile under PHP 7, right?